### DISCUSSION TITLE
Why do my images don't load in production when using URL params?

### DISCUSSION THREAD

**Discussion by florianwalther-private** [vercel/next.js#39183] [Help]
I use [react-markdown](https://github.com/remarkjs/react-markdown) to parse markdown and replace `img` tags for `next/image`s. To set the width and height of the image, I add them as URL search params in markdown like this:

```
![Calculator App](/images/blog/calulcator_app.png?width=700&height=394)
```

I then **parse** these params with react-markdown and set the image dimensions accordingly. The code for that should not be relevant to the question because it works. In **development mode**, the images are displayed in the correct size: 

![image](https://user-images.githubusercontent.com/30524875/181906653-8c045d17-2086-48b3-b551-73b33b30430f.png)

However, in **production**, even tho the image containers have the correct size as well, the **URL of the image seems to be broken**:

![image](https://user-images.githubusercontent.com/30524875/181906702-7772e1b8-fd86-4d8a-8d58-1caaffbce12f.png)
 
**Why are my images not loading properly in production?** Does NextJS not like URL params in image src URLs? If so, why does it work in dev mode?

**Comment by florianwalther-private** [comment]
I tried passing the src without the URL params and it still doesn't load. So weird. The image does exist on the server tho.

**Comment by florianwalther-private** [comment]
Ok, the problem was caused by the images having an uppercase `.PNG` as their file extensions. I renamed it to lowercase `.png` and now it works. Why does Next case about this (only in production)?

**Reply by GabenGar** [reply]
Because the dev server doesn't care about it while the prod server does.

**Reply by florianwalther-private** [reply]
Hm, yea I just wonder why. Well, I'm happy that I found the problem!
